trainingcore/src/de/hybris/training/core/converter/ManufacturerImpexConverter.java

```diff
@@ -0,0 +1,58 @@
+package de.hybris.training.core.converter;
+
+import de.hybris.platform.acceleratorservices.dataimport.batch.converter.impl.DefaultImpexConverter;
+import de.hybris.platform.servicelayer.exceptions.SystemException;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Required;
+import org.springframework.util.Assert;
+
+import java.util.Calendar;
+import java.util.Map;
+
+public class ManufacturerImpexConverter extends DefaultImpexConverter {
+    private static final char PLUS_CHAR = '+';
+    private String impexRow;
+
+    protected void processValues(final Map<Integer, String> row, final StringBuilder builder, final int idx, final int endIdx)
+    {
+        final boolean mandatory = impexRow.charAt(idx + 1) == PLUS_CHAR;
+        Integer mapIdx = null;
+        try
+        {
+            mapIdx = Integer.valueOf(impexRow.substring(mandatory ? idx + 2 : idx + 1, endIdx));
+        }
+        catch (final NumberFormatException e)
+        {
+            throw new SystemException("Invalid row syntax [invalid column number]: " + impexRow, e);
+        }
+        final String colValue = row.get(mapIdx);
+        if(mapIdx.equals(3)){
+            String isOld="false";
+            int establishmentYear= Integer.parseInt(row.get(mapIdx-1));
+            int CurrentYear= Calendar.getInstance().get(Calendar.YEAR);
+            if(CurrentYear-establishmentYear>=10){
```

Comment:
Remove the hardcoded value "10" and make it as a configurable. In future if we have to change 10 to 15 or 10 to 8 we will be easily do it with out any code changes